pkgs/native_assets_cli/test/model/asset_test.dart

```diff
@@ -156,16 +157,16 @@ void main() {
 
   test('asset yaml', () {
     final yaml = yamlEncode([
-      for (final item in assets) item.toYaml(BuildOutputImpl.latestVersion)
+      for (final item in assets) item.toJson(BuildOutputImpl.latestVersion)
```

Comment:
So the asset list still is a YAML?

Reply:
No, this test is testing what if we encode the JSON object (not string) to YAML. I find that easier to test.

Reply:
That's personal preference then :) I find JSON to be very readable (if nicely formatted).